### PR TITLE
Limit issues displayed on contributor page

### PR DESCRIPTION
**Pull request by kedromelon**
Added feature mentioned in https://github.com/babel/babel.github.io/pull/1192#issuecomment-284830515

(will gladly take on the other bullet point later too!)

### CODE REVIEW

**Comment by kedromelon** [comment]
also -- what should the limit be? 5 seemed good to me, but perhaps that's a little low.

**Comment by tkgogogo** [comment]
https://github.com/babel/babel.github.io/pull/1192#issuecomment-284830515

**Comment by tkgogogo** [comment]
https://localhost:50443/#Scratch

**Comment by kedromelon** [comment]
added filtering of beginner issues out from the "help wanted" issues list

also changed callbacks to filter/limit data via lodash -- i think this might be more readable? let me know if tweaks need to be made to better fit code style here though :)

File: scripts/contributing.js [hunk 1]
```diff
@@ -36,9 +38,11 @@
                 labels: 'beginner-friendly'
             },
             success: function(data){
-                for(var i = 0; i < data.length; i++){
-                    $('.open-issues-section--beginner .issues').append('<li class="issue"><a href="' + data[i].html_url + '"><span class="issue__number">#' + data[i].number +'</span><span class="issue__title">' + data[i].title + '</span></a></li>');
-                }
+                var limitedData = _.take(data, ISSUES_LIMIT);
+
+                _.forEach(limitedData, function(issue) {
```
Comment (Daniel15) [inline scripts/contributing.js]: What's the advantage of `_.forEach` over `limitedData.forEach`?

File: scripts/contributing.js [hunk 2]
```diff
@@ -36,9 +38,11 @@
                 labels: 'beginner-friendly'
             },
             success: function(data){
-                for(var i = 0; i < data.length; i++){
-                    $('.open-issues-section--beginner .issues').append('<li class="issue"><a href="' + data[i].html_url + '"><span class="issue__number">#' + data[i].number +'</span><span class="issue__title">' + data[i].title + '</span></a></li>');
-                }
+                var limitedData = _.take(data, ISSUES_LIMIT);
```
Comment (Daniel15) [inline scripts/contributing.js]: `data.slice(0, ISSUES_LIMIT)`

File: scripts/contributing.js [hunk 3]
```diff
@@ -36,9 +38,11 @@
...
+
+                _.forEach(limitedData, function(issue) {
+                    $('.open-issues-section--beginner .issues').append('<li class="issue"><a href="' + issue.html_url + '"><span class="issue__number">#' + issue.number +'</span><span class="issue__title">' + issue.title + '</span></a></li>');
```
Comment (Daniel15) [inline scripts/contributing.js]: I know this is similar to how it was before, but ideally we should remove the XSS vulnerabilities from this 😛 

We have jQuery available here, so this should **not** be building HTML elements like this:
```
<span class="issue__title">' + issue.title + '</span>
```

File: scripts/contributing.js (hunk 1 above)
Comment (kedromelon) [inline scripts/contributing.js]: hm, i suppose none. at least, nothing intentional. i see no reason to use lodash over it -- i'll change it

File: scripts/contributing.js (hunk 3 above)
Comment (kedromelon) [inline scripts/contributing.js]: great point! i'll change that here

**Comment by tkgogogo** [comment]
1dccffa

**Comment by kedromelon** [comment]
gonna close this -- it's sat for a while and is trivial enough to be implemented elsewhere easily. feel free to reopen if needed though!

**Comment by hzoo** [comment]
Hey Noah, would totally like to add something like this again for the new site

and if we could find a way to link a label to a page that would be cool

**Pull request closed without merging**